app/src/components/market/sections/market_create/steps/outcomes/index.tsx

```diff
@@ -210,6 +212,12 @@ const Outcomes = (props: Props) => {
     }
   }
 
+  const onOutcomeNameChange = (value: string) => {
+    setNewOutcomeName(value)
+    const val = value.toLowerCase();
+    outcomes.some(i => val.indexOf(i.name.toLowerCase()) >= 0) ? setDuplicateOutcome(true) : setDuplicateOutcome(false)
```

Comment:
This could be clearer, but is just an opinion, something like
```
const isDuplicated = outcomes.map(o => o.name.toLowerCase()).includes(value.toLowerCase())
setDuplicateOutcome(isDuplicated)
```